application/views/bestitamazonpay4oxid_paybutton.tpl

```diff
@@ -0,0 +1,130 @@
+[{capture name="sBestitAmazonPayButton"}]
+
+    [{*Older Amazon Pay functionality !*}]
+    [{if $oViewConf->getAmazonPayIsActive() && !$oViewConf->getAmazonLoginIsActive() && !$smarty.session.amazonOrderReferenceId}]
+        [{assign var="sAmazonWidgetUrl" value=$oViewConf->getAmazonProperty('sAmazonWidgetUrl')}]
+        [{assign var="sAmazonSellerId" value=$oViewConf->getAmazonConfigValue('sAmazonSellerId')}]
+        [{assign var="sModuleUrl" value=$oViewConf->getModuleUrl('bestitamazonpay4oxid')}]
+
+        [{oxscript include="`$sAmazonWidgetUrl`?sellerId=`$sAmazonSellerId`" priority=11}]
+        [{oxscript include="`$sModuleUrl`out/src/js/bestitamazonpay4oxid.js" priority=11}]
+        [{oxstyle  include="`$sModuleUrl`out/src/css/bestitamazonpay4oxid.css"}]
+
+        <div id="payWithAmazonDiv" class="amazonContentGroup">
+            <div class="amazonTooltip">
+                <i>?</i>
+                <div class="amazonTooltipContent">[{oxmultilang ident="BESTITAMAZONPAY_PAY_BUTTON_HINT"}]</div>
+            </div>
+            <div id="payWithAmazonButton">
+                <img src="[{$oViewConf->getAmazonProperty('sAmazonButtonUrl')}]?sellerId=[{$oViewConf->getAmazonConfigValue('sAmazonSellerId')}]&size=x-large" title="[{ oxmultilang ident='BESTITAMAZONPAY_PAY_WITH_AMAZON_BUTTON'}]"/>
```

Comment:
Unnecessary whitespace at **[{ oxmultila**ng ident='BESTITAMAZONPAY_P....

- [ ] Please remove the whitespace

Reply:
The code of the button was just moved to an other file so I didn't any refactoring.